packages/create-block-tutorial-template/templates/$slug.php.mustache

```diff
@@ -26,7 +26,7 @@
  * Behind the scenes, it registers also all assets so they can be enqueued
  * through the block editor in the corresponding context.
  *
- * @see https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/writing-your-first-block-type/
+ * @see https://developer.wordpress.org/block-editor/reference-guides/block-api/block-registration/
```

Comment:
This link points now to the JavaScript API for block registration. We should use either https://developer.wordpress.org/reference/functions/register_block_type/ or something more higher-level like maybe https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/.

Reply:
Good suggestion, I updated to the PHP function used. 👍